### PR TITLE
[Merged by Bors] - Cleanup many sprites stress tests

### PR DESCRIPTION
**Pull request by nicopap**
Since the new renderer, no frustum culling is applied to 2d components
(be it Sprite or Mesh2d), the stress_tests docs is therefore misleading
and should be updated.

Furthermore, the `many_animated_sprites` example, unlike `many_sprites`
kept vsync enabled, making the stress test less useful than it could be.
We now disable vsync for `many_animated_sprites`.

Also, `many_animated_sprites` didn't have the stress_tests warning
message, instead, it had a paragraph in the module doc. I replaced the
module doc paragraph by the warning message, to be more in line with
other examples.

## Solution

- Remove the paragraph about frustum culling in the `many_sprites`
  and `many_animated_sprites` stress tests

### CODE REVIEW

**Review by devil-ira — APPROVED** [review]

**Comment by rparrett** [comment]
Are there no longer plans for adding 2d frustum culling?

There's a stalled out PR here: https://github.com/bevyengine/bevy/pull/3944. It seems potentially worth doing.

**Comment by nicopap** [comment]
@rparrett Does make sense. Furthermore #3944 seems both a clear win and still fairly easy to implement. I'll reword it.

File: examples/stress_tests/many_animated_sprites.rs [hunk 1]
```diff
@@ -37,6 +43,8 @@ fn setup(
     assets: Res<AssetServer>,
     mut texture_atlases: ResMut<Assets<TextureAtlas>>,
 ) {
+    warn!(include_str!("warning_string.txt"));
```
Comment (alice-i-cecile) [inline examples/stress_tests/many_animated_sprites.rs:46]: Where is this getting imported from / why does it exist? Even reading the PR description I couldn't quite piece it together.

Comment (rparrett) [reply examples/stress_tests/many_animated_sprites.rs:46]: This was added to other stress tests here: https://github.com/bevyengine/bevy/pull/5029

**Comment by alice-i-cecile** [comment]
@nicopap CI appears to be failing due to a broken import. Once that's fixed, I'm happy to merge this.

File: examples/stress_tests/many_animated_sprites.rs (hunk 1 above)
Comment (nicopap) [reply examples/stress_tests/many_animated_sprites.rs:47]: That's what I was referring to in the third paragraph of the PR description ("to be more in line with other examples").

**Pull request closed without merging**